I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script_aa_WeekLoop.rpy", line 33, in script
    $ ARS_typography = ARS_typography + 1
  File "game/script_aa_WeekLoop.rpy", line 33, in <module>
    $ ARS_typography = ARS_typography + 1
NameError: name 'ARS_typography' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script_aa_WeekLoop.rpy", line 33, in script
    $ ARS_typography = ARS_typography + 1
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\ast.py", line 881, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\python.py", line 1913, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script_aa_WeekLoop.rpy", line 33, in <module>
    $ ARS_typography = ARS_typography + 1
NameError: name 'ARS_typography' is not defined

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
SBU Senior Project 1.0
Wed Jan 30 23:40:27 2019
